apps/scorecard/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "upperair",
```

Comment:
This needs to be scorecard, not upperair.

Reply:
good catch. I changed and pushed this.